support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.jsx

```diff
@@ -225,7 +225,7 @@ const paper = (countryGroupId: CountryGroupId, priceCopy: PriceCopy, isTop: bool
 //       }),
 //     }],
 //     productImage: <PaperAndDigitalPackshot />,
-//     offer: getSaleCopy(PaperAndDigital, countryGroupId).bundle.subHeading,
+//     offer: '',
```

Comment:
Could we delete all this commented code?

Reply:
This is the code for the Print and Digital section on the main subs landing page, which I believe we decided to hold onto for the time being.